rust/operator-binary/src/main.rs

```diff
@@ -58,21 +60,57 @@ async fn main() -> anyhow::Result<()> {
         Command::Run(ProductOperatorRun {
             product_config,
             watch_namespace,
-            tracing_target,
+            telemetry_arguments,
             cluster_info_opts,
         }) => {
-            stackable_operator::logging::initialize_logging(
-                "AIRFLOW_OPERATOR_LOG",
-                APP_NAME,
-                tracing_target,
-            );
-            stackable_operator::utils::print_startup_string(
-                crate_description!(),
-                crate_version!(),
-                built_info::GIT_VERSION,
-                built_info::TARGET,
-                built_info::BUILT_TIME_UTC,
-                built_info::RUSTC_VERSION,
+            let _tracing_guard = Tracing::builder()
+                // TODO (@Techassi): This should be a constant
+                .service_name("airflow-operator")
```

Comment:
Should we use `built_info::PKG_NAME` here? The Cargo.tomls seem to have fairly suitable names?

Reply:
We'll look at this in a follow up PR.